app/pdfs/response_report.rb

```diff
@@ -0,0 +1,99 @@
+class ResponseReport < Prawn::Document
+  def initialize(response)
+    super(page_size: 'A4')
+    @response = response
+
+    register_font_family
+    font 'DejaVu Sans'
+
+    header_text 'Inleiding'
+    introduction
+    move_down 20
+
+    creation_date
+    move_down 20
+
+    header_text 'Resultaten'
+    domain_results_table
+    move_down 20
+
+    header_text 'Antwoorden'
+    answers_table
+  end
+
+  private
+
+  def register_font_family
+    font_path = File.join(Rails.root.to_s, 'vendor', 'fonts')
+    font_families.update(
+      'DejaVu Sans' => {
+        normal: File.join(font_path, 'DejaVuSans.ttf'),
+        italic: File.join(font_path, 'DejaVuSans-Oblique.ttf'),
+        bold: File.join(font_path, 'DejaVuSans-Bold.ttf'),
+        bold_italic: File.join(font_path, 'DejaVuSans-BoldOblique.ttf')
+      }
+    )
+  end
+
+  def header_text(str)
+    text str, size: 24, style: :bold
+  end
+
+  def creation_date
+    text "Ingevuld op #{I18n.l @response.created_at, format: :long}", style: :bold_italic
+  end
+
+  def domain_results
+    @response.domain_ids.each do |domain_id|
+      text "Domein: #{domain_id}"
+    end
+  end
+
+  def domain_results_table
+    table_data = @response.domain_results.map do |domain_result|
+      domain = domain_result.domain
+      [domain.description, domain.norm_population, domain_result.quartile, domain_result.estimate_interpretation]
+    end
+
+    table table_data, column_widths: [150, 150, 100, 100]
+  end
+
+  def answers_table
+    table_data = @response.questions.map do |question|
+      question_text = question.intro.blank? ? question.text : "<i>#{question.intro}</i><br/>#{question.text}"
```

Comment:
De logica hier is bedoeld om de eventueel aanwezige intro in italic te tonen (plus een linebreak). Die logica past wat mij betreft wel in een view.